Sources/ProjectSpec/Scheme.swift

```diff
@@ -31,6 +33,25 @@ public struct Scheme: Equatable {
         self.profile = profile
         self.analyze = analyze
         self.archive = archive
+        self.management = management
+    }
+
+    public struct Management: Equatable {
+        public static let sharedDefault = true
+
+        public var shared: Bool
+        public var orderHint: Int?
+        public var isShown: Bool?
+
+        public init(
+            shared: Bool = true,
```

Comment:
Can we use `sharedDefault`

Reply:
You mean this right? 

```
shared: Bool = Scheme.Management.sharedDefault
```

Reply:
done https://github.com/yonaskolb/XcodeGen/pull/1142/commits/8edccc68abe50f44399480bff6757b48b99734db